framework/test/jest/specs/integration/example.spec.js

```diff
@@ -0,0 +1,5 @@
+describe('A simple test', () => {
+	it('should pass', () => {
+		expect(2 + 4).toBe(6);
+	});
+});
```

Comment:
Can we add some skeleton of a real example and implement few tests cases for this PR.

Reply:
@nazarhussain I'm already working on adding real test examples on #2951. In this PR just focused on configuring the test runner.